includes/SkinWMAU.php

```diff
@@ -6,17 +6,6 @@
  * @ingroup Skins
  */
 class SkinWMAU extends SkinMustache {
-
-	/**
-	 * Initialise the page.
-	 * @param OutputPage $out
-	 */
-	public function initPage( OutputPage $out ) {
-		$out->addMeta( 'viewport', 'width=device-width, initial-scale=1.0' );
```

Comment:
The viewport meta tag is no longer being output with this patch. Is that a 1.35 problem?

Reply:
hmm yeh looks like it.

Timeless does the following...
```
 if ( version_compare( $wgVersion, '1.36', '<' ) ) {
                        // Make responsive - this is replaced by `responsive` option in 1.36
                        $out->addMeta( 'viewport',
                                'width=device-width, initial-scale=1.0, ' .
                                'user-scalable=yes, minimum-scale=0.25, maximum-scale=5.0'
                        );
```


https://phabricator.wikimedia.org/T258290 introduced it, in case it makes sense to backport that to 1.35 to aid adoption...